docs/MO_DG/prepare_model/customize_model_optimizer/Customize_Model_Optimizer.md

```diff
@@ -6,217 +6,218 @@
    :maxdepth: 1
    :hidden:
 
+   openvino_docs_MO_DG_prepare_model_customize_model_optimizer_Customize_Model_Optimizer_Model_Optimizer_Ports_Connections
+   openvino_docs_MO_DG_prepare_model_customize_model_optimizer_Model_Optimizer_Extensions
    openvino_docs_MO_DG_prepare_model_customize_model_optimizer_Extending_Model_Optimizer_With_Caffe_Python_Layers
 
-@endsphinxdirective
-
-
 This article describes Model Optimizer internals. Altering them may result in application instability, and in case of future changes to the API, lack of backward compatibility.
```

Comment:
I don't understand why we are investing to modifying and better organizing internal part of the documentation for the legacy MO infrastructure? With migrating to the new Tensorflow frontend we don't need to propose extensibility framework based on old MO extensions for users. Everything will be moved to the new extensions described in a dedicated part. This one should be deprecated soon. Please prepare a move of this whole document to developer documentation with appropriate legacy statement rather then keeping and updating it here. The main document for extensibility that everybody should extend and contribute to is https://docs.openvino.ai/latest/openvino_docs_Extensibility_UG_Frontend_Extensions.html. It works for both MO and runtime, this is a single extension API for OpenVINO. The only thing that prevented us from more aggressive removing of this legacy part was Tensorflow. Now we are migrating to the new one and this legacy part is obsolete.